src/main/kotlin/com/petqua/application/OauthService.kt

```diff
@@ -0,0 +1,19 @@
+package com.petqua.application
```

Comment:
oauth 패키지로 분리해도 좋아보여요

```suggestion
com.petqua.application.oauth
```